bin/runtime/src/lib.rs

```diff
@@ -262,7 +262,7 @@ parameter_types! {
     // follows polkadot : https://github.com/paritytech/polkadot/blob/9ce5f7ef5abb1a4291454e8c9911b304d80679f9/runtime/polkadot/src/lib.rs#L369
     pub const OperationalFeeMultiplier: u8 = 5;
     // We expect that on average 25% of the normal capacity will be occupied with normal txs.
-    pub const TargetSaturationLevel: Perquintill = Perquintill::from_percent(25);
+    pub const TargetSaturationLevel: Perquintill = Perquintill::from_percent(50);
```

Comment:
There's also a comment above that is outdated now.

Reply:
fixed,  thanks for catching this